pom.xml

```diff
@@ -423,7 +423,9 @@
             </plugin>
 
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.22.1</version>
```

Comment:
This version is specified in the super-pom (10-RC2) so I don't think is needed here anymore.